package.json

```diff
@@ -92,7 +92,7 @@
     "@nrwl/angular": "^9.1.2",
     "@types/prismjs": "1.9.0",
     "firebase": "6.2.4",
-    "highcharts": "7.0.3",
+    "highcharts": "^8.0.4",
```

Comment:
Should this be pinned to specific version?
```suggestion
    "highcharts": "8.0.4",
```

Reply:
I think not, it is pinned in the lock file. Minor+patch upgrades should be non breaking.